package.json

```diff
@@ -68,6 +68,6 @@
   "resolutions": {
     "typescript": "^5.3.3"
   },
-  "version": "0.9.3-1rc4",
+  "version": "0.9.3-1r6",
```

Comment:
technically, this is not needed now, while making the release we can change it.

Reply:
Okay